_posts/2018-11-16-webhooks-about.markdown

```diff
@@ -27,16 +34,24 @@ The top-level attributes are webhook specific, and an example would look somethi
   "entity_type": "Employee",
   "entity_id": 1123581321345589,
   "timestamp": 1533606328,
-  "entity_attributes": { }
+  "entity_attributes": { },
+  "partner_attributes": { }
 }
 ```
 
-In this example, `timestamp`, `event_type`, and `entity_attributes` are all standard keys to have in a webhooks payload. In Gusto's system permissions are based on an resource and it's entities. The parent reference is the `resource` while the `entity` is the reference to what actually triggered the event.
+In this example, `timestamp`, `event_type`, and `entity_attributes` are all standard keys to have in a webhooks payload.
+In Gusto's system permissions are based on an resource and it's entities. The parent reference is the `resource` while
+the `entity` is the reference to what actually triggered the event.
+
+Additionally, any events of type `*.provisioned` may have a `partner_attributes` key. If Gusto has a mapping of that
+entity to an entity in your application, those attributes will contain your identifier to that entity. See the 
+[Partner Attributes](/v1/partner_attributes) section for more information on its contents.
```

Comment:
This part is new, the rest is just fixing spacing (markdown allows single line breaks without breaking the line in the output).

Reply:
Actually, I don't care for this whole section. I think another pass at this would help things out, maybe replacing it with a params table would be better.

Reply:
+1 for params table with a sample payload and sample response (similar to API docs)

Reply:
@eschmidt Sounds good. I've now written this up, but would prefer to make it a separate PR to cultivate more specific discussion.

Mind accepting this PR if you don't have any other concerns?

Reply:
Just have the outstanding nit regarding a/an.

Reply:
Not worth holding up the review though (esp if another PR is coming soon), so I will accept.

Reply:
Ah, I fixed that locally and forgot to push :)